lib/matplotlib/cbook/deprecation.py

```diff
@@ -249,11 +249,15 @@ def wrapper(*args, **kwargs):
         old_doc = inspect.cleandoc(old_doc or '').strip('\n')
 
         message = message.strip()
-        new_doc = ('[*Deprecated*] {old_doc}\n'
-                   '\n'
-                   '.. deprecated:: {since}\n'
-                   '   {message}'
-                   .format(since=since, message=message, old_doc=old_doc))
+        # Add "Deprecated" to top of docstring
+        new_doc = '[*Deprecated*] {old_doc}\n\n'.format(old_doc=old_doc)
+        # Add a notes section if one isn't already present
+        note_section = 'Notes\n-----'
```

Comment:
Does this need the same leading `\n` that the header in `__init__` has?

Reply:
I think I might have put an extra `\n` in the `new_doc` line just above, but probably makes sense to move it here.